apps/nextjs-contentful/README.md

```diff
@@ -69,7 +69,7 @@ After setting up your environment, you can run the application in development or
 To start the development server, which allows for live reloading and other development features, run the following command:
 
 ```bash
-npm run start
+npm run start:nextful
```

Comment:
IMO we should continue using just `start` to run the "main app" and append whatever we need to run the other apps